fastify.js

```diff
@@ -707,3 +707,11 @@ function loadVersion () {
 module.exports = fastify
 module.exports.fastify = fastify
 module.exports.default = fastify
+
+try {
+  const dc = require('diagnostics_channel')
+  dc.channel('fastify:loaded').publish(fastify)
+} catch (e) {
```

Comment:
```suggestion
} catch {
```

^ supported back to at least Node 10. So there's no issue with this syntax for any current or future version of Fastify.